Source/Scene/ImplicitSubtree.js

```diff
@@ -108,19 +108,46 @@ Object.defineProperties(ImplicitSubtree.prototype, {
       return this._metadataExtension;
     },
   },
+
+  /**
+   * Gets the implicit tile coordinates for the root of the subtree.
+   *
+   * @type {ImplicitTileCoordinates}
+   * @readonly
+   * @private
+   */
+  implicitCoordinates: {
+    get: function () {
+      return this._implicitCoordinates;
+    },
+  },
 });
 
 /**
- * Check if a specific tile is available
+ * Check if a specific tile is available at an index
  *
  * @param {Number} index the index of the desired tile
  * @returns {Boolean} the value of the i-th bit
  * @private
  */
-ImplicitSubtree.prototype.tileIsAvailable = function (index) {
+ImplicitSubtree.prototype.tileIsAvailableAtIndex = function (index) {
   return this._tileAvailability.getBit(index);
 };
 
+/**
+ * Check if a specific tile is available at an implicit tile coordinate
+ *
+ * @param {ImplicitTileCoordinates} implicitCoordinates The coordinates of a tile
+ * @returns {Boolean} the value of the i-th bit
+ * @private
+ */
+ImplicitSubtree.prototype.tileIsAvailableAtCoordinates = function (
```

Comment:
Wanted the ability to query by coordinates as well. This way external systems don't need to know what morton encoding is

Reply:
Yeah these additions/changes to the interface make sense now that we have more requirements for implicit tiling.